vignettes/articles/study_sdtm_spec.Rmd

```diff
@@ -0,0 +1,361 @@
+---
+title: "All about Metadata"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{All about Metadata}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+```{css, echo = FALSE}
+#pkgdown-sidebar{
+ display:none;
+}
+.wide .table{
+  font-size: 8px;
+  overflow: visible
+}
+
+```
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+options(rmarkdown.html_vignette.check_title = FALSE)
+```
+
+{sdtm.oak} is a metadata-driven solution that is designed to be Electronic Data 
+Capture (EDC) and standards agnostic. Throughout this article, the term "metadata" 
+is used several times. In this context, "metadata" refers to the specific 
+metadata used by {sdtm.oak}. This article aims to provide users with a more 
+detailed understanding of the {sdtm.oak} metadata.
+
+In general, metadata can be defined as "data about data." It does not include 
+any patient-level data. Instead, the metadata provides a blueprint of the data 
+that needs to be collected during a study.
+
+## Standards Metadata
+
+The standards metadata used in {sdtm.oak} is sourced from the CDISC Library or 
+sponsor MDR or any other form of documentation where standards are maintained. 
+This metadata provides information on the following:
+
+- The relationship between Data Collection Standards (eCRF & eDT), SDTM mapping, 
+and Controlled Terminology
+- Machine-readable standard SDTM mappings
+- Algorithms and associated metadata required for the SDTM automation of 
+standards in the study.
+
+In the upcoming releases of {sdtm.oak}, we will effectively utilize the 
+standards metadata and customize it to meet the study requirements.
+
+## Study Definition Metadata
+
+Study Definition Metadata is also referred to as Study Metadata. Study Definition 
+Metadata provides information about the eCRF and eDT data collected in the study.
+
+**eCRF Metadata**
+The eCRF Design Metadata is fetched from the EDC system. This Metadata includes
+
+- Forms Metadata: Identifier, eCRF label, Repeating format and other properties of the eCRF.
+
+- Fields Metadata: Identifier, question label, datatype, and other 
+properties of data collection fields in the study.
+
+- Data Dictionaries: Identifier and the controlled terms collected at the
+source.
+
+- Visits: Name of the visits as defined in the EDC.
+
+**eDT Metadata**
+
+eDT Metadata is the blueprint metadata that describes the data collected as part of that external data transfer. This includes
+
+- Dataset name, label, repeating properties, etc.
+
+- Variable name, datatype, label and associated codelist, etc.
```

Comment:
Does eDT metadata describe the eCRF metadata? Can it be completely derived from it? Or does it have some unique information?